recipes/test_processes.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+#
+# Cookbook Name:: aws-parallelcluster
+# Recipe:: tests_processes
+#
+# Copyright 2013-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
minor: year is wrong everywhere

Reply:
Will fix in another PR as I've just seen this error in many files. Thanks